codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -165,9 +162,14 @@ public void execute() throws MojoExecutionException {
                     logger.info("Generating java file for YAML file [" + dataFile.getName() + "] in domain ["
                             + domain + "]");
                     
-                    String tempPackage = basePackage;
+                    String tempPackage = "";
+                    // add the basePackage, iff it contains a value
+                    if (!basePackage.isEmpty()) {
+                        tempPackage += basePackage;
```

Comment:
The if test is not strictly necessary now.  The direct "tempPackage += basePackage;" now handles both cases.